specification/customer-insights/resource-manager/Microsoft.CustomerInsights/2017-01-01/customer-insights.json

```diff
@@ -203,7 +207,17 @@
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The name of the hub."
+            "description": "The name of the Hub.",
+            "pattern": "^[a-zA-Z][a-zA-Z0-9]+$",
+            "minLength": 1,
+            "maxLength": 64
+          },
+          {
+            "name": "organizationUrl",
```

Comment:
Where is this used?  There is a semantic validation error due to this, see the [log](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/286291712).

Reply:
Fixed this. The "organizationUrl" should be in the query segment of the URL